django_app/redbox_app/jinja2.py

```diff
@@ -95,6 +95,9 @@ def environment(**options):
             "to_user_timezone": to_user_timezone,
             "environment": settings.ENVIRONMENT.value,
             "security": settings.MAX_SECURITY_CLASSIFICATION.value,
+            "repo_owner": settings.REPO_OWNER,
```

Comment:
I'd prefer we used waffle as this is our standards for feature flagging
https://waffle.readthedocs.io/en/stable/